PinpointKit/PinpointKit/Sources/PinpointKit.swift

```diff
@@ -41,7 +41,8 @@ public class PinpointKit {
      - parameter delegate:           A delegate that is notified of significant events.
      */
     public convenience init(feedbackRecipients: [String], delegate: PinpointKitDelegate? = nil) {
```

Comment:
Thanks for raising it. I think the convenience makes it worth it, and the static function you suggest (or `let` perhaps) rids us of enough risk to them falling out of sync.